### PR TITLE
FIXED: README.md code sample in "'ref' style querying" section.

### PR DESCRIPTION
**Pull request by ctrlplusb**
Running the current example:

```
let Greeting = props => <div>hello <strong _testID='name'>{props.name}</strong></div>;

$(Greeting).render()
  .prop({ name: 'Betty' })
  .find('[_testID=name]')
  .text()
  .should.equal('Betty')
```

Results in the following error:

```
Invariant Violation: ReactDOM.render(): Invalid component element. Instead of passing a component class, make sure to instantiate it by passing it to React.createElement.
      at invariant (node_modules/fbjs/lib/invariant.js:39:15)
      at Object.ReactMount._renderSubtreeIntoContainer (node_modules/react/lib/ReactMount.js:505:89)
      at Object.ReactMount.render (node_modules/react/lib/ReactMount.js:570:23)
      at Object.wrapper [as render] (node_modules/react/lib/ReactPerf.js:66:21)
      at render (node_modules/teaspoon/utils.js:101:40)
      at QueryCollection.render (node_modules/teaspoon/element.js:110:20)
      at Context.<anonymous> (_test.js:25:19)
```

The initial fix for this (component -> element) is:

```
let Greeting = props => <div>hello <strong _testID='name'>{props.name}</strong></div>;

$(<Greeting />).render()
  .prop({ name: 'Betty' })
  .find('[_testID=name]')
  .text()
  .should.equal('Betty')
```

But this still produces the following error:

```
TypeError: (0 , _teaspoon2.default)(...).render(...).prop is not a function
      at Context.<anonymous> (_test.js:26:10)
```

I eventually guessed the following, which works:

```
let Greeting = props => <div>hello <strong _testID='name'>{props.name}</strong></div>;

$(<Greeting />)
  .props({ name: 'Betty' })
  .render()
  .find('[_testID=name]')
  .text()
  .should.equal('Betty')
```

### CODE REVIEW

**Comment by ctrlplusb** [comment]
Related to #15

File: README.md [hunk 1]
```diff
@@ -253,8 +253,9 @@ component's "real" props. In this example lets use `_testID`.
 ```js
 let Greeting = props => <div>hello <strong _testID='name'>{props.name}</strong></div>;
 
-$(Greeting).render()
-  .prop({ name: 'Betty' })
+$(<Greeting />)
```
Comment (jquense) [inline README.md]: thanks, tho I think the thing would be to do `$(<Greeting name='Betty' />)` for the example as its a bit simpler

Comment (ctrlplusb) [inline README.md]: Too true. :)  I'll make that change.

I do think there exists some confusion about the .prop vs .props still though.  Might be worth reviewing the other sections too.

Comment (jquense) [inline README.md]: Ya I just fixed that in the readme so it shouldn't be a problem I hope

Comment (ctrlplusb) [inline README.md]: Great, sorted my side too.

Thanks for this awesome library.  I've been waiting a long time for this to come along.

**Comment by jquense** [comment]
thanks for the help!

**Pull request merged**